moon/docker/rover/run_solution.bash

```diff
@@ -41,12 +41,12 @@ ROBOT_PIDS=()
 ROS_PIDS=()
 DAEMON_PIDS=()
 
-echo "Start robot solution"
+echo "Start robot solution (no timeout)"
 export OSGAR_LOGS=`pwd`
 cd /osgar
 for s in ${JSONFILES[@]}; do
     echo "starting recording of $s"
-    python3 -m osgar.record --duration 2700 moon/config/$s --note "collect some ROS data" &
+    python3 -m osgar.record moon/config/$s --note "collect some ROS data" &
```

Comment:
well, I should also mention the darkside ... it will run now infinitely, i.e. the log file will be very large (it could kill my system). Also I am not sure if the bulk test will still work? Is NASA at least killing the ROS master after 45 minutes (simulated)?
Alternative would be to request shutdown internally after simulated time??

Reply:
Bulk test has its own timeout and kills the docker at the end. If you run it interactively, presumably you kill it yourself when done. Not sure in what usage scenario it would run forever, or at least until it clogs the computer. We can check what happens after 45 mins sim time with the submission process, gazebo definitely gets killed.